pyopia/statistics.py

```diff
@@ -302,7 +302,8 @@ def make_montage(stats_file, pixel_size, roidir,
     makes nice looking montage from a directory of extracted particle images
 
     Args:
-        stats_file                  : location of the stats hdf5 or nc file that comes from processing
+        stats                       : either a str specifying the location of the STATS.nc file that comes from processing,
```

Comment:
Suggest rename "stats_file" input to something like "stats_file_or_df", and change the documentation line to match.